Makefile

```diff
@@ -198,7 +198,8 @@ dockerrun: dockerbuild-debug
 
 .PHONY: clean_venv
 clean_venv:
-	pipenv --rm
+	# ignore pipenv errors by adding command prefix -
+	-pipenv --rm
```

Comment:
what kind of errors are these? is the env still removed or do we have to verify that it's actually removed?

Reply:
Yes it happens if the environment has already been removed, then the following clean step are not done because `pipenv --rm` act as `rm -r` and not as `rm -rf`